service/storage_service.hh

```diff
@@ -507,7 +507,7 @@ public:
      * @param ranges
      * @return mapping of ranges to the replicas responsible for them.
     */
-    std::unordered_map<dht::token_range, inet_address_vector_replica_set> construct_range_to_endpoint_map(
+    std::unordered_map<dht::token_range, inet_address_vector_replica_set> construct_range_to_endpoint_map_in_thread(
```

Comment:
btw, we can make construct_range_to_endpoint_map_in_thread private too.

Reply:
We can even kill it  because it is used only once. It is also a trivial wrapper.

Reply:
Agreed

Reply:
Done in v2